docs/interpreter/jdbc.md

```diff
@@ -269,38 +269,76 @@ To develop this functionality use this [method](http://docs.oracle.com/javase/7/
  </table>
 
 ### Phoenix
-#### Properties
+ Phoenix supports 'thick' and 'thin' connection types:
+
+ The thick client is faster, but must connect directly to ZooKeeper and HBase RegionServers.
+
+ The thin client has fewer dependencies and connects through a [Phoenix Query Server](http://phoenix.apache.org/server.html) instance.
+
+ Use the appropriate phoenix.driver and phoenix.url for your connection type.
+
```

Comment:
Hi @randerzander. Thanks for your contribution!

Probably it's a nitpick, currently those 4 lines are rendered like below, 
![screen shot 2016-09-20 at 4 16 59 pm](https://cloud.githubusercontent.com/assets/10060731/18660858/b82ee1dc-7f4d-11e6-8786-16277a747aa8.png)


So how about changing the 4 lines to like this? (You can just copy&paste if you want :D )
```
Phoenix supports `thick` and `thin` connection types:

 - Thick client is faster, but must connect directly to ZooKeeper and HBase RegionServers.
 - Thin client has fewer dependencies and connects through a [Phoenix Query Server](http://phoenix.apache.org/server.html) instance.

Use the appropriate `phoenix.driver` and `phoenix.url` for your connection type.
```

Then it will be rendered 
![screen shot 2016-09-20 at 4 17 14 pm](https://cloud.githubusercontent.com/assets/10060731/18660895/f0eef7dc-7f4d-11e6-99da-a4099866c733.png)